geomstats/geometry/heisenberg.py

```diff
@@ -1,12 +1,13 @@
 """Implementation of the 3D Heisenberg group."""
 
 import geomstats.backend as gs
+from geomstats.geometry.base import VectorSpace
 from geomstats.geometry.euclidean import Euclidean
 from geomstats.geometry.lie_group import LieGroup
 from geomstats.geometry.symmetric_matrices import SymmetricMatrices
 
 
-class HeisenbergVectors(LieGroup):
+class HeisenbergVectors(LieGroup, VectorSpace):
     """Class for the 3D Heisenberg group in the vector representation.
 
     The 3D Heisenberg group represented as R^3. It is a step-2 Carnot Lie
```

Comment:
NIT: Actually, the parameters of the init should be added, instead of "No parameters" in the class' docstring

Reply:
But there are no parameters for the init either!